engineering_tools/xil-cloud-formation-v0.yaml

```diff
@@ -543,7 +543,40 @@ Resources:
           chmod +x /home/ubuntu/.xsession
           chown ubuntu:ubuntu /home/ubuntu/.xsession
 
+          # Add NVIDIA package repositories
+          distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+          sudo apt-get update
```

Comment:
Might be good to remove this sudo apt-get update since we do it again at line 553